runners/direct-java/src/test/java/org/apache/beam/runners/direct/DirectGraphVisitorTest.java

```diff
@@ -95,8 +95,8 @@ public void getRootTransformsContainsRootTransforms() {
     assertThat(graph.getRootTransforms(), hasSize(3));
     assertThat(
         graph.getRootTransforms(),
-        Matchers.containsInAnyOrder(
-            graph.getProducer(created), graph.getProducer(counted), graph.getProducer(unCounted)));
+        Matchers.containsInAnyOrder(new Object[] {
```

Comment:
Is the usage of new Object[] really required?

Reply:
As far as I can tell, there are two options that work:
1) use new Object[] { ....  }
2) cast each parameter to either (Object) or (AppliedPTransform) without the type params.   The latter then introduces rawtype warnings.